### ISSUE TITLE
[New Format]: Inscopix

### ISSUE THREAD

**Issue by CodyCBakerPhD** [catalystneuro/neuroconv#707]
### What format would you like to see added to NeuroConv?

Inscopix finally releases an open API

### Does the format have any documentation?

Not as far as I can tell, but could be inferred from some of the source code at https://github.com/inscopix/py_isx/blob/main/isx/__init__.py

### Existing APIs for format

https://github.com/inscopix/py_isx

### Do you have any example files you are willing to share?

Test files found with releases: https://github.com/inscopix/py_isx/releases

### Do you have any interest in helping implement the feature?

Yes.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/catalystneuro/neuroconv/blob/main/.github/CODE_OF_CONDUCT.rst)
- [X] Have you ensured this bug was not already [reported](https://github.com/catalystneuro/neuroconv/issues)?

**Comment by bendichter** [comment]
Example data: https://github.com/inscopix/py_isx/releases

**Comment by bendichter** [comment]
@CodyCBakerPhD see test data addition PR here: https://gin.g-node.org/CatalystNeuro/ophys_testing_data/pulls/19

**Comment by bendichter** [comment]
blocked by https://github.com/catalystneuro/roiextractors/issues/274

**Comment by CodyCBakerPhD** [comment]
You mean blocked by https://github.com/inscopix/py_isx/issues/45?

**Comment by CodyCBakerPhD** [comment]
Oh I see, https://github.com/catalystneuro/roiextractors/issues/274 blocked by https://github.com/inscopix/py_isx/issues/45 as a chain

**Comment by bendichter** [comment]
I'm thinking about it as a chain of dependencies. This is blocked by https://github.com/catalystneuro/roiextractors/issues/274 and https://github.com/catalystneuro/roiextractors/issues/274 is blocked by https://github.com/inscopix/py_isx/issues/45, so effectively this is blocked by https://github.com/inscopix/py_isx/issues/45. I wish GitHub offered a better way to express this formally.

**Comment by CodyCBakerPhD** [comment]
Do they not even have a static FPS attributed to the files? Exact timestamps technically aren't necessary for instantiating the SegmentationExtractor

**Comment by bendichter** [comment]
Yes but it looks like they have not exposed that attribute yet in a non-private method

**Comment by sg-s** [comment]
also docs, are available here: https://inscopix.github.io/py_isx/

**Comment by h-mayorquin** [comment]
Documentation link: https://inscopix.github.io/pyisx/index.html

@daphnedequatrebarbes  has been  moving forward to add segmentation and imaging interfaces for this format (see #1361 and #1364). This is great and probably the most important.

The following is a to-do list to be done at some point depending on priorities:

- [ ] Imaging: add support for multi-channel (see `nVue` [system](https://inscopix.com/nvue-system/))
- [ ] Imaging: add support for automatically determining one-photon vs two-photon (see `nVista 2P` [system](https://inscopix.com/nvista-2p-system/))
- [ ] Imaging: Add support for multi-plane (see [link](https://github.com/catalystneuro/neuroconv/pull/1361#discussion_r2153283039) here for discussion)
- [ ] Add an interface for adding events (`EventSet`)
- [ ] Add an interface for analog signals (`GPIO`)
- [ ] Add an interface for inertial measurements (`IMU`)
- [ ] Add an interface for behavioral movie (`MicrosopeMovie`)
- [ ] Add a converter that adds all of the above.

See the list of the supported types in the documentation:
https://inscopix.github.io/pyisx/overview.html#file-types

**Comment by daphnedequatrebarbes** [comment]
another thing to consider : 

- [ ] Segmentation : add emission/excitation wavelengths fixed for each microscope (see isx [PR](https://github.com/inscopix/pyisx/issues/62))
